docs/changes/20171027-organization-users.md

```diff
@@ -0,0 +1,5 @@
+## Organization Users
```

Comment:
are we updating this?

Reply:
no idea, i would tag @@support too so they know how to get organization users from each organization

Reply:
I think we should

Reply:
Posted in flow